src/v2/modules/documents/routes.js

```diff
@@ -33,6 +33,23 @@ exports.getDocuments = {
   }
 };
 
+exports.getDocumentByRefAndDate = {
+  method: 'GET',
+  path: '/crm/2.0/document/search',
```

Comment:
Following my comments in the water service PR i suggest we change the url to /crm/2.0/documents/search.

Reply:
I have no problem with this, but the ticket wanted a single document result to be returned, which is why I did the route as `/document` rather than `/documents`.

Are you sure you would like us to reuse the existing path? This will return an array of objects.

Reply:
Linked to this https://github.com/DEFRA/water-abstraction-tactical-crm/pull/609#discussion_r483460457